config/database.php

```diff
@@ -59,7 +59,7 @@
             'strict' => true,
             'engine' => null,
             'options' => extension_loaded('pdo_mysql') ? array_filter([
-                PDO::MYSQL_ATTR_SSL_CA => env('MYSQL_ATTR_SSL_CA'),
+                (class_exists(\Pdo\Mysql::class) ?\Pdo\Mysql::ATTR_SSL_CA : \PDO::MYSQL_ATTR_SSL_CA) => env('MYSQL_ATTR_SSL_CA'),
```

Comment:
Adjusted `config/database.php` to use `Pdo\Mysql::ATTR_SSL_CA` when available (fallback to `PDO::MYSQL_ATTR_SSL_CA`) to avoid deprecation notices on newer PHP versions. This doesn’t affect the in-memory SQLite test setup used by `php artisan test`.